pkg/master/services.go

```diff
@@ -21,28 +21,23 @@ import (
 	"net"
 
 	"k8s.io/klog"
-	kubeoptions "k8s.io/kubernetes/pkg/kubeapiserver/options"
 	"k8s.io/kubernetes/pkg/registry/core/service/ipallocator"
 )
 
-// DefaultServiceIPRange takes a the serviceIPRange flag and returns the defaulted service ip range (if  needed),
-// api server service IP, and an error
-func DefaultServiceIPRange(passedServiceClusterIPRange net.IPNet) (net.IPNet, net.IP, error) {
-	serviceClusterIPRange := passedServiceClusterIPRange
-	if passedServiceClusterIPRange.IP == nil {
-		klog.Infof("Network range for service cluster IPs is unspecified. Defaulting to %v.", kubeoptions.DefaultServiceIPCIDR)
-		serviceClusterIPRange = kubeoptions.DefaultServiceIPCIDR
+// GenericAPIServerServiceIP takes a the serviceIPRange flag and returns the api server service IP, and an error
+func GenericAPIServerServiceIP(serviceClusterIPRange net.IPNet) (net.IP, error) {
```

Comment:
this isn't generic, it only applies to kube-apiserver, possibly just rename to `ServiceIPRange`?